vaux-client/src/client.rs

```diff
@@ -449,8 +562,7 @@ impl MqttClient {
                                                 .is_err()
                                                 {
                                                     // TODO handle the pub ack next time through
-                                                    // push a message to the last error channel
-                                                    eprintln!("unable to send puback");
+                                                    // push a message to the last error channel\
```

Comment:
Was this a TODO basically? Also is the trailing \ an error?

Reply:
open TODO to be resolve in future handling on PUBACK